src/components/Header/Header.tsx

```diff
@@ -1,22 +1,22 @@
-import { FC } from 'react'
```

Comment:
Let's separate stylistic changes from logical changes to make the review size smaller.

I noticed recently there is inconsistent semicolon usage. We can talk about whether we want the [semi](https://eslint.org/docs/latest/rules/semi) rule on or off, I've heard good arguments for both cases.